.travis.yml

```diff
@@ -0,0 +1,31 @@
+# Mac OS X Python build
+# admin access required
+sudo:
+  required
+# operating system
+os:
+  osx
+# language used by the os
+language:
+  objective-c
+# cache directories
+cache:
+  directories:
+    - $HOME/.cache/pip3
+# clear cache log files
+before_cache:
+  - rm -f $HOME/.cache/pip3/log/debug.log
+python:
+- "3.6"
+before_install:s
```

Comment:
was that `s` a typo?

Reply:
I guess it should just be `before_install:`

Reply:
lol yeah it was 😄